llvm/examples/OrcV2Examples/LLJITWithRemoteDebugging/LLJITWithRemoteDebugging.cpp

```diff
@@ -173,24 +174,15 @@ int main(int argc, char *argv[]) {
     TSMs.push_back(ExitOnErr(parseExampleModuleFromFile(Path)));
   }
 
-  StringRef TT;
+  std::string TT;
   StringRef MainModuleName;
   TSMs.front().withModuleDo([&MainModuleName, &TT](Module &M) {
     MainModuleName = M.getName();
     TT = M.getTargetTriple();
+    if (TT.empty())
+      TT = sys::getProcessTriple();
   });
 
-  for (const ThreadSafeModule &TSM : TSMs)
```

Comment:
Why does this need to be removed? Will the program fail gracefully if a module has a triplet that is not supported by the built LLVM?

Reply:
When I added the example, I used it for experimentation with multiple IR inputs and I ran into a bug once that was related to a triple mismatch. That's why it checked the triples of all input files upfront. We could fix it to work with triple-less modules, but I didn't have the time today and I don't think it's worth it actually.